ui/src/views/mint/github-step/github-repository-selection.tsx

```diff
@@ -0,0 +1,156 @@
+import {
+  Button,
+  Card,
+  Combobox,
+  ComboboxItem,
+  Flex,
+  Grid,
+  Icon,
+  IconButton,
+} from '@/components';
+import { Input } from '@/components/core/input';
+import { Separator } from '@/components/core/separator.styles';
+import React, { forwardRef, useRef, useState } from 'react';
+import { Mint } from '../mint.context';
+
+//TODO remove once it's integrated with GH login
+const repos = [

```

Comment:
Same idea from the previous component, we need to be aware with loading states and how it will be fetched from github api. Also other thing to notice is handling the error scenarios.